packages/sources/src/components/button/button.scss

```diff
@@ -380,8 +380,25 @@ $dso-button-tertiary-active-color: $scampi;
   }
 }
 
+@mixin button_map_focus() {
+  $focus-map: (
+    "background-color": $dso-button-secondary-hover-background-color,
```

Comment:
Ik heb ze bewust aan de `-hover-` variables gehangen, ik wil expliciet maken dat de focus styling gelijk is aan de hover styling